src/styles/blocks/header.scss

```
@@ -0,0 +1,70 @@
+.header {
+  padding: 0 50px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  margin: 0;
+  background: #fff;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+  min-width: 700px;
+
+  &__logo {
+    font-size: 0;
+  }
+}
+
+.nav {
+  display: flex;
+  align-items: center;
+
+  &__list {
+    display: flex;
+  }
+
+  &__item {
+    list-style: none;
+
+    &:not(:last-child) {
+      margin-right: 20px;
+    }
+  }
+
+  &__link {
+    font-size: 12px;
+    text-transform: uppercase;
+    text-decoration: none;
+    color: var(--main-accent);
```

```suggestion
    color: var(--main-accent);
```

Use scss variables. Fix all cases